modules/richards/tests/gravity_head_2/tests

```diff
@@ -171,4 +171,43 @@
     rel_err = 1e-5
     abs_zero = 1e-4
   [../]
+
+  [./except01]
+    type = 'RunException'
+    input = 'gh01.i'
+    cli_args = 'Materials/rock/viscosity="1"'
+    expect_err = 'There are 2 Richards fluid variables, so you need to specify this number of viscosities, relperm_UO, seff_UO, sat_UO, density_UO, SUPG_UO'
+  [../]
+  [./except02]
+    type = 'RunException'
+    input = 'gh01.i'
```

Comment:
Bum - too many PRs confusing my brain.  I'm not sure how to remove the Outputs block using cli_args.  Can you give me an example?